listenbrainz/spark/request_manage.py

```diff
@@ -269,30 +271,12 @@ def request_similar_users(max_num_users):
 @cli.command(name='cron_request_all_stats')
 @click.pass_context
 def cron_request_all_stats(ctx):
-    ctx.invoke(request_user_stats, type_="entity", range_="week", entity="artists")
-    ctx.invoke(request_user_stats, type_="entity", range_="month", entity="artists")
-    ctx.invoke(request_user_stats, type_="entity", range_="year", entity="artists")
-    ctx.invoke(request_user_stats, type_="entity", range_="all_time", entity="artists")
-
-    ctx.invoke(request_user_stats, type_="entity", range_="week", entity="releases")
-    ctx.invoke(request_user_stats, type_="entity", range_="month", entity="releases")
-    ctx.invoke(request_user_stats, type_="entity", range_="year", entity="releases")
-    ctx.invoke(request_user_stats, type_="entity", range_="all_time", entity="releases")
-
-    ctx.invoke(request_user_stats, type_="entity", range_="week", entity="recordings")
-    ctx.invoke(request_user_stats, type_="entity", range_="month", entity="recordings")
-    ctx.invoke(request_user_stats, type_="entity", range_="year", entity="recordings")
-    ctx.invoke(request_user_stats, type_="entity", range_="all_time", entity="recordings")
-
-    ctx.invoke(request_user_stats, type_="listening_activity", range_="week")
-    ctx.invoke(request_user_stats, type_="listening_activity", range_="month")
-    ctx.invoke(request_user_stats, type_="listening_activity", range_="year")
-    ctx.invoke(request_user_stats, type_="listening_activity", range_="all_time")
-
-    ctx.invoke(request_user_stats, type_="daily_activity", range_="week")
-    ctx.invoke(request_user_stats, type_="daily_activity", range_="month")
-    ctx.invoke(request_user_stats, type_="daily_activity", range_="year")
-    ctx.invoke(request_user_stats, type_="daily_activity", range_="all_time")
+    for stats_range in ALLOWED_STATISTICS_RANGE:
```

Comment:
this does all types for a range and then moves on to the next range, as opposed to what was before when it was all ranges for a type. Does it matter in which order we do it?

Reply:
Yes, I mentioned the same in the commit message. I don't think we care.